Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -248,24 +244,50 @@ public KeyCommand()
         }
 
         private void Add(
-            PrivateKey key,
+            string key,
             PassphraseParameters passphrase,
             bool json,
-            bool dryRun
+            bool dryRun,
+            string pathString,
+            bool create
         )
         {
             string passphraseValue = passphrase.Take("Passphrase: ", "Retype passphrase: ");
-            ProtectedPrivateKey ppk = ProtectedPrivateKey.Protect(key, passphraseValue);
-            Guid keyId = dryRun ? Guid.NewGuid() : KeyStore.Add(ppk);
-            if (json)
+            string path = Path.GetFullPath(pathString);
+            if (create)
             {
-                using Stream stdout = System.Console.OpenStandardOutput();
-                ppk.WriteJson(stdout, keyId);
-                stdout.WriteByte(0x0a);  // line ending
+                PrivateKey pkey = new PrivateKey();
+                ProtectedPrivateKey ppk = ProtectedPrivateKey.Protect(pkey, passphraseValue);
+                Guid keyId = dryRun ? Guid.NewGuid() : KeyStore.Add(ppk);
+                if (json)
+                {
+                    FileStream fs = File.Create(path);
+                    ppk.WriteJson(fs, keyId);
+                    fs.WriteByte(0x0a);
+                    fs.Close();
+                }
             }
             else
             {
-                PrintKeys(new[] { (keyId, ppk) });
+                if (json)
+                {
+                    if (File.Exists(path))
+                    {
+                    ProtectedPrivateKey ppk = ProtectedPrivateKey.FromJson(File.ReadAllText(path));
+                    Guid keyId = dryRun ? Guid.NewGuid() : KeyStore.Add(ppk);
```

Comment:
```suggestion
                        ProtectedPrivateKey ppk = ProtectedPrivateKey.FromJson(File.ReadAllText(path));
                        Guid keyId = dryRun ? Guid.NewGuid() : KeyStore.Add(ppk);
```
Just for coding style agreement in out products :)